packages/alpine/os_lnx.h.patch

```diff
@@ -1,11 +1,16 @@
---- ./imap/src/osdep/unix/os_lnx.h.orig	2020-11-04 19:47:00.079064606 +0000
-+++ ./imap/src/osdep/unix/os_lnx.h	2020-11-04 19:48:23.228984716 +0000
-@@ -58,6 +58,9 @@
+diff -Naur ./alpine.orig/imap/src/osdep/unix/os_lnx.h ./alpine.patch/imap/src/osdep/unix/os_lnx.h
+--- ./imap/src/osdep/unix/os_lnx.h	2021-05-16 06:44:05.008507235 +0000
++++ ./imap/src/osdep/unix/os_lnx.h	2021-05-16 06:44:57.139105846 +0000
+@@ -58,6 +58,13 @@
  
  #define direct dirent
  
++#ifndef L_SET
```

Comment:
Hmm yes, i've tried again it built just fine.
I added that addition were due to an error of redefinition of these consts, i added that wrap so that it will compile and more safer than deleting the patch.

That package built using my own fork of the builder which patched all termux prefixes variables, maybe that's the cause. Right now i'm using offical builder to test.

Anyway I think this addition cant get things worsen and is safe (no-op for valid build environment), keep it will help me from the need of maintaining a fork, but if you're rejecting it's ok, i'll remove this patch.

Reply:
That's unlikely that prefix change resulted in build error, at least in this case. However if you have changed the API level, this can happen.

Official Termux API level is 24. We do not support API levels below this value and dropped all related patches when deprecated Android 5.x / 6.x.

Reply:
Yes i think it's unlikely too, i'll remove this patch.